File: 1512-NumberOfGoodPairs/1512-NumberOfGoodPairs.cpp
// Last updated: 9/10/2026, 9:53:29 AM
class Solution {
public:
    int numIdenticalPairs(vector<int>& nums) {
        int s=nums.size();
        int sum=0;
        for(int i=0;i<s;i++){
            for(int j=i+1;j<s;j++){
                if(nums[i]==nums[j]){
                    sum+=1;
                }
            }
        }
        return sum;
    }
};

File: 1512-NumberOfGoodPairs/1512-NumberOfGoodPairs.py
# Last updated: 9/10/2026, 9:53:35 AM
class Solution(object):
    def numIdenticalPairs(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        l=[]
        for i in range(len(nums)):
            for j in range(i+1,len(nums)):
                if (nums[i]==nums[j]):
                    l.append((i,j))
        return len(l)
        